astroquery/gaia/core.py

```diff
@@ -243,7 +470,8 @@ def query_object(self, coordinate, radius=None, width=None, height=None,
         The job results (astropy.table).
         """
         return self.__query_object(coordinate, radius, width, height,
-                                   async_job=False, verbose=verbose, columns=columns)
```

Comment:
we switched to allow longer line length (100 is desirable, but 120 is also OK in case e.g. long strings),  no need to wrap this line